index.html

```diff
@@ -50,6 +50,189 @@ <h1 class="intro height">I’m Kartikay Glad to see you!</h1>
+              sign-ups required.
+            </p>
+            <ul class="tech-stack">
+              <li><button class="tech-items" type="button">html</button></li>
+              <li><button class="tech-items" type="button">css</button></li>
+              <li><button class="tech-items" type="button">javascript</button></li>
+            </ul>
+            <a class="card-button" href="#">See Project</a>
+          </div>
+        </div>
+        
+        <div class="card multipost">
+  
+          <div class="card-element">
+            <div class="img-container multipost">
+              <img class="work-image" src="./Images/Multiple-two.png" alt="work-Image" />
+            </div>
+            <h2 class="card-title">
+              Multi-Post Stories
+            </h2>
+            <div class="card-section">
+              <p class="para-one">CANOPY</p>
+              <i class="fa fa-circle"></i>
+              <p class="para-two">Back End Dev</p>
+              <i class="fa fa-circle"></i>
+              <p class="para-two">2015</p>
+            </div>
+            <p class="card-body">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="tech-stack">
+              <li><button class="tech-items" type="button">html</button></li>
+              <li><button class="tech-items" type="button">css</button></li>
+              <li><button class="tech-items" type="button">javascript</button></li>
+            </ul>
+            <a class="card-button" href="#">See Project</a>
+          </div>
+        </div>
+  
+      </div>
+      <div class="about-sec">
+        <div class="about-content-container">
+          <h2 class="about-intro">About<br />
+            Myself
+          </h2>
+          <p class="about-description">
+            Hello I’m a software developer! I can help you build a product,
+            feature or website. Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
+          </p>
+          <a class="bottom-connect about-connect">LET’S CONNECT</a>
+          <ul class="icon-container about-icon">
+            <li>
+              <a class="social-icon" href="#"><img src="./Images/Github-icon.png" alt="Github" /></a>
+            </li>
+            <li>
+              <a class="social-icon" href="#"><img src="./Images/Linkedin-icon.png" alt="Linkedin" /></a>
+            </li>
+            <li>
+              <a class="social-icon" href="#"><img src="./Images/Angellist-icon.png" alt="Angellist" /></a>
+            </li>
+            <li>
+              <a class="social-icon" href="#"><img src="./Images/Twitter-icon.png" alt="Twitter" /></a>
+            </li>
+            <li>
+              <a class="social-icon" href="#"><img src="./Images/Medium-icon.png" alt="Medium" /></a>
+            </li>
```

Comment:
- [ ] [Optional] Nicely done 👍, you have a pretty nice, clean, and organized code, in your future projects to avoid the assets management you could make use of icons galleries, there are a lot of them around on internet, and there are a lot of benefits from using them, 2 that are my favorite ones is [FontAwesome](https://fontawesome.com/) and [Google Icons](https://fonts.google.com/icons), here you can find for example [LinkedIn icon](https://fontawesome.com/search?p=1&q=linked%20in&s=solid%2Cbrands)